control_file_gen.pl

```diff
@@ -1325,6 +1325,8 @@ ()
    #
    # create run description
    $rundesc = "cs:$csdate"."0000 cy:$nhcName$advisorynum ASGS";
+   # create the RUNID
+   $ensembleid = $addHours . " hour " . $enstorm . " run";
```

Comment:
Thank you, I did not know that. Please feel free to reject in the future ... getting the code into the right shape is the most important priority. :-)

Reply:
The other consideration in existing scripts is what is done through out. In other words, the inconsistency would have actually bothered me whereas this didn't bother me at all. It was just an FYI. Especially when it comes to existing stuff, there's value in doing something consistently throughout. So that's the other consideration - changing it would have come with the temptation to "change them all" :)

Reply:
Understood ... is there a list of best practices for coding (maybe in the wiki) where we can add idiomatic changes like this ? We can make an issue to "change 'em all" (one file at a time) ... or at least make me more of best practices for new scripts ...